Specifications/Events.Store.MongoDB/events.cs

```diff
@@ -1,31 +1,31 @@
 // Copyright (c) Dolittle. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
+using Dolittle.Runtime.Events.Store.Streams;
 using MongoDB.Bson;
 
 namespace Dolittle.Runtime.Events.Store.MongoDB
 {
     public static class events
     {
-        public const string some_event_content = "{\"something\": \"text\"}";
+        public const string some_event_content = "{ \"something\" : \"text\" }";
 
         public static BsonDocument some_event_content_bson_document => BsonDocument.Parse("{\"something\": \"text\"}");
 
-        public static event_builder new_event(uint event_log_sequence_number, uint aggregate_version) => new event_builder(event_log_sequence_number, aggregate_version);
+        public static event_builder new_event(EventLogSequenceNumber event_log_sequence_number, AggregateRootVersion aggregate_version) => new event_builder(event_log_sequence_number, aggregate_version);
```

Comment:
I would call this `new_event_from_aggregate` just so it is a bit more clear in the specs.